packages/pilot/src/pages/ErrorBoundary/reducer.js

```diff
@@ -0,0 +1,197 @@
+import {
+  append,
+  concat,
+  defaultTo,
+  difference,
+  equals,
+  head,
+  ifElse,
+  includes,
+  is,
+  pipe,
+  prop,
+  reduce,
+  reject,
+  test,
+  uncurryN,
+  unless,
+  when,
+  where,
+} from 'ramda'
+
+import {
+  API_ERROR_RECEIVE,
+  CLEAR_ALL_ERRORS,
+  CLEAR_ERROR,
+  REACT_ERROR_RECEIVE,
+} from './actions'
+import getResponseError from '../../formatters/apiError'
+import getReactError from '../../formatters/reactError'
+import localErrors from '../../errors'
+
+const initialState = []
+
+const addNewError = uncurryN(2, err => unless(
+  includes(err),
+  append(err)
+))
+
+const removeError = uncurryN(2, err => when(
+  includes(err),
+  reject(equals(err))
+))
+
+const removeRouteQuery = route =>
```

Comment:
Isso pode ficar na mesma linha
```suggestion
const removeRouteQuery = route => head(route.split('?'))
```